WKT2_TO_PROJJSON.md

```diff
@@ -0,0 +1,204 @@
+# WKT2 to PROJJSON Converter
```

Comment:
Thank you for explaining the method in detail. Please remove this markdown file from the source tree, and add it as a comment in the PR instead.